packages/angular/testing/src/testing-utils.ts

```diff
@@ -18,6 +20,8 @@ export function readAll<T>(o: Observable<T>): Promise<T[]> {
 }
 
 /**
+ * @deprecated This will be removed in a later version of Nx. Since RxJS 7, use firstValueFrom(obs$)
```

Comment:
Same as previous.

Reply:
I don't think this one should be either. We want the first value in this case as the utility is `readFirst`

Reply:
ups, I didn't even read the method name 😅